modules/imaging_browser/test/TestPlan.md

```diff
@@ -1,44 +1,43 @@
 ## Imaging Browser test plan
 	
-### A. Imaging Browser front page
-1. User can load Imaging Browser module front page IFF has permission imaging_browser_view_site, imaging_browser_view_allsites, imaging_browser_phantom_allsites or imaging_browser_phantom_ownsite [Partial Automation Testing; To ADD phantom permissions automated tests]
+### Imaging Browser main page
+1. User can access Imaging Browser module front page IFF has permission imaging_browser_view_site, imaging_browser_view_allsites, imaging_browser_phantom_allsites or imaging_browser_phantom_ownsite [Partial Automation Testing; To ADD phantom permissions automated tests]
 2. User can see other sites Imaging datasets IFF has permission imaging_browser_view_allsites. User can see only own site Imaging datasets IFF has permission imaging_browser_view_site. User can see phantom data only from across all sites with the imaging_browser_phantom_allsites, and from own sites with imaging_browser_phantom_ownsite [Partial Automation Testing; TO ADD phantom permissions automated tests]
-3. Test that all Filters work.  Upon first loading, Site filter should be set to 'All User Sites' (with the dropdown menu populated with the sites the logged on user is affiliated with) IFF user does not have permission imaging_browser_view_allsites nor imaging_browser_phantom_allsites, and is set to 'All' otherwise.
+3. Test that all filters work.
+    - When the Site filter is empty, all sites with which the user is associated should be displayed. Every site should be displayed if the user has the `imaging_browser_view_allsites` permission.
 4. Test Clear Filters button
 5. Test column table is sortable by headers
-6. Test that Links work, to correct dataset (selected/native)
-7. Add more modalities (from the Scan_type column of the mri_scan_type table) to the Configuration/Imaging Modules/Tabulated Scan Types field, and ensure that for each added modality, a new corresponding column shows up in the Imaging Browser table    
+6. Ensure that the hyperlinks in the Links column are active and load the correct dataset.
+7. Add more modalities (from the Scan_type column of the `mri_scan_type` table) to the Configuration/Imaging Modules/Tabulated Scan Types field, and ensure that for each added modality, a new corresponding column shows up in the Imaging Browser table. (This requires back-end access.)
 
-### B. ViewSession / Volume List
+### View Session / Volume List
```

Comment:
not sure if that was your intent for this section but i dont like it

![Screenshot from 2020-04-08 18-06-44](https://user-images.githubusercontent.com/15268287/78838276-d2c60d80-79c3-11ea-9e94-ce3a0b29f8f3.png)

Reply:
Nope, not my intention. I'll fix the formatting.

Reply:
@johnsaigle I'm not sure why the numbering was removed ?